### DISCUSSION TITLE
У всех дс не работает сейчас?

### DISCUSSION THREAD

**Discussion by skiddgoddamn** [bol-van/zapret#840] [Q&A]
Ютуб работает, дс откис, хотя пару часов назад все было ок, поспрашивал друзей которые юзали запрет, говорят тоже не работает, что делать? Мб прикрыли лавочку нам.😢😢😢

**Comment by mar4uello** [comment]
у меня тоже не робит, у девушки тоже. ток ютуб работает

**Comment by skiddgoddamn** [comment]
Пишите сюда если найдете фикс пожалуйста

**Comment by opgray** [comment]
тоже не робит

**Comment by WutADude** [comment]
Провайдер не ЭР-Телеком (Домру)?

**Comment by skiddgoddamn** [comment]
БизнесСвязь провайдер

**Comment by miniSenjor** [comment]
У меня не работает, у друзей работает

**Comment by rekharo** [comment]
+, тоже не работает
при этом с ютубом всё ок

**Comment by vlkhv0x** [comment]
Во время игры отвалился дискорд, балдеж

**Comment by Dellayned** [comment]
на дом.ру откисло с кайфом наглухо вообще

**Comment by de1ju** [comment]
есть решение проблемы?

**Comment by skiddgoddamn** [comment]
> есть решение проблемы?

Ждем

**Comment by 5empa1** [comment]
Енева(спб) тоже отвал

**Comment by daniyolka** [comment]
Мне с кентами помогло это:

Новый архив качаете (https://github.com/bol-van/zapret-win-bundle), распак, в папке zaptet-winws: 

запустил от админа killall (остановить службы)
запустил от админа task_remove (убрать старую службу)

в cmd от админа:

sc delete windivert
sc stop windivert
sc qc windivert

в папке zapret-winws:
запускаю zapret-winws/preset_russia.cmd из нового архива ОТ АДМИНА И НЕ ЗАКРЫВАТЬ ОКНО!!!!


**upd. От 01.11.24**
Ниже в треде подсказали решение, чтобы окно не держать, и оно помогло:
Создаете текстовик, пишете туда этот текст, тип файла в .bat, запуск от админа - работает как раньше, окно это держать не нужно.

```
set ARGS=--wf-tcp=80,443 --wf-udp=443,50000-50099 ^
--filter-udp=50000-50099 --ipset="%~dp0ipset-discord.txt" --dpi-desync=fake --dpi-desync-repeats=6 --dpi-desync-any-protocol --dpi-desync-cutoff=n2 --new ^
--filter-udp=50000-50099 --new ^
--filter-udp=443 --hostlist="%~dp0list-youtube.txt" --dpi-desync=fake --dpi-desync-repeats=11 --dpi-desync-fake-quic="%~dp0quic_initial_www_google_com.bin" --new ^
--filter-udp=443 --dpi-desync=fake --dpi-desync-repeats=11 --new ^
--filter-tcp=80 --dpi-desync=fake,split2 --dpi-desync-autottl=2 --dpi-desync-fooling=md5sig --new ^
--filter-tcp=443 --hostlist="%~dp0list-youtube.txt" --dpi-desync=fake,split2 --dpi-desync-repeats=11 --dpi-desync-fooling=md5sig --dpi-desync-fake-tls="%~dp0tls_clienthello_www_google_com.bin" --new ^
--dpi-desync=fake,disorder2 --dpi-desync-autottl=2 --dpi-desync-fooling=md5sig

set SRVCNAME=zapret

net stop "%SRVCNAME%"
sc delete "%SRVCNAME%"
sc create "%SRVCNAME%" binPath= ""%~dp0winws.exe" %ARGS%" DisplayName= "zapret winws1" start= auto
sc description "%SRVCNAME%" "zapret"
sc start "%SRVCNAME%"
```

**Comment by opgray** [comment]
можете ссылку кинуть на новый архив
а то у меня нет такой папки и проги

**Comment by skiddgoddamn** [comment]
Сделай git clone https://github.com/bol-van/zapret.git

**Comment by skiddgoddamn** [comment]
> Мне с кентами помогло это:
> 
> Новый архив качаете, распак, в папке zaptet-winws:
> 
> запустил от админа killall (остановить службы) запустил от админа task_remove (убрать старую службу)
> 
> в cmd от админа:
> 
> sc delete windivert sc stop windivert sc qc windivert
> 
> в папке zapret-winws: запускаю zapret-winws/preset_russia.cmd из нового архива ОТ АДМИНА И НЕ ЗАКРЫВАТЬ ОКНО!!!!

щас тестану, отпишу если что сюда

**Comment by WutADude** [comment]
Помогло коверканье протокола TCP по хостам дискорда как в случае с ютубом, с последующим старым коверканьем UDP.

Конкретно проблема была с запуском самого дискорда и подключением на каналы.

**Comment by dananana11** [comment]
Тоже я хз чё делать

**Comment by kusbxd** [comment]
Способ который помог большинству моих друзей и мне.

https://github.com/bol-van/zapret-win-bundle
- качайте это, заходите в папку "**zapret-winws**" и запускаете **preset_russia**

Дальше должно всё воркать как раньше.

**Comment by skiddgoddamn** [comment]
> Новый архив качаете, распак, в папке zaptet-winws:
> 
> запустил от админа killall (остановить службы) запустил от админа task_remove (убрать старую службу)
> 
> в cmd от админа:
> 
> sc delete windivert sc stop windivert sc qc windivert
> 
> в папке zapret-winws: запускаю zapret-winws/preset_russia.cmd из нового архива ОТ АДМИНА И НЕ ЗАКРЫВАТЬ ОКНО!!!!

Эта тема сработала, спасибо

**Comment by skiddgoddamn** [comment]
с этими командами была проблема отказано в доступе, но и без них сработало:

sc delete windivert
sc qc windivert

**Comment by lisakit42** [comment]
работает да, но держать открытым файл нууу так себе, поэтому если найдете вариант для фонового процесса, дайте знать :)

**Comment by dananana11** [comment]
О дс заработал

**Comment by TheDoukiDrift** [comment]
Даров, всё ещё не робит, при этом до момента удаления службы (показывал другу, т.к. у него первого отвалился дс)  всё работало, как поставил службу повторно, всё умерло.
P.S. у меня имено fork для дс, который я чуть сократил для знакомых, если чё могу аргументы скинут в bat-нике.

**Comment by TheDoukiDrift** [comment]
Домены если чё все теже
Могу скинуть список

**Comment by TheDoukiDrift** [comment]
Ну у меня так выглядит fork 
![image](https://github.com/user-attachments/assets/37db2e22-ec71-480d-add2-7d198f384341)

**Comment by dananana11** [comment]
у меня войс не работает

**Comment by TheDoukiDrift** [comment]
А у меня впринципе дс не работает, интересно, проблемы в zapret или опять РКН с провайдерами фигню мутит

**Comment by PETYASEIL** [comment]
а как скачать этот файл

**Comment by limpopo34** [comment]
> Мне с кентами помогло это:
> 
> Новый архив качаете, распак, в папке zaptet-winws:
> 
> запустил от админа killall (остановить службы) запустил от админа task_remove (убрать старую службу)
> 
> в cmd от админа:
> 
> sc delete windivert sc stop windivert sc qc windivert
> 
> в папке zapret-winws: запускаю zapret-winws/preset_russia.cmd из нового архива ОТ АДМИНА И НЕ ЗАКРЫВАТЬ ОКНО!!!!

У меня почему-то нет killall в папке, поэтому я просто остановил сервис, сделал всё то что было до запуска пресета и всё равно ютуб почему-то работает. Походу я не смог с этой рекомендацией остановить службы, ну и после запуска пресета ничего не поменялось (ну и окно которое вы просите не закрывать закрывается сразу же само собой)

**Comment by PETYASEIL** [comment]
как скачать этот запрет из гитхаба?

**Comment by skiddgoddamn** [comment]
> > Мне с кентами помогло это:
> > Новый архив качаете, распак, в папке zaptet-winws:
> > запустил от админа killall (остановить службы) запустил от админа task_remove (убрать старую службу)
> > в cmd от админа:
> > sc delete windivert sc stop windivert sc qc windivert
> > в папке zapret-winws: запускаю zapret-winws/preset_russia.cmd из нового архива ОТ АДМИНА И НЕ ЗАКРЫВАТЬ ОКНО!!!!
> 
> У меня почему-то нет killall в папке, поэтому я просто остановил сервис, сделал всё то что было до запуска пресета и всё равно ютуб почему-то работает. Походу я не смог с этой рекомендацией остановить службы, ну и после запуска пресета ничего не поменялось (ну и окно которое вы просите не закрывать закрывается сразу же само собой)

Сделай git clone https://github.com/bol-van/zapret.git
Или скачай архив [тут](https://github.com/bol-van/zapret/releases/tag/v66)
В папке по поиску ищи

**Comment by Rakajlnik** [comment]
> Провайдер не ЭР-Телеком (Домру)?

я так понимаю если да - сидим кайфуем?

**Comment by pepper-forms** [comment]
кто придумал как сервисом сделать эту штуку поделитесь

**Comment by xd-ridik-xd** [comment]
вот бы по оффису РКН ядеркой жахнуть...

**Comment by tesum** [comment]
Чтобы сделать сервисом: 

set ARGS=--wf-tcp=80,443 --wf-udp=443,50000-50099 ^
--filter-udp=50000-50099 --ipset="%~dp0ipset-discord.txt" --dpi-desync=fake --dpi-desync-repeats=6 --dpi-desync-any-protocol --dpi-desync-cutoff=n2 --new ^
--filter-udp=50000-50099 --new ^
--filter-udp=443 --hostlist="%~dp0list-youtube.txt" --dpi-desync=fake --dpi-desync-repeats=11 --dpi-desync-fake-quic="%~dp0quic_initial_www_google_com.bin" --new ^
--filter-udp=443 --dpi-desync=fake --dpi-desync-repeats=11 --new ^
--filter-tcp=80 --dpi-desync=fake,split2 --dpi-desync-autottl=2 --dpi-desync-fooling=md5sig --new ^
--filter-tcp=443 --hostlist="%~dp0list-youtube.txt" --dpi-desync=fake,split2 --dpi-desync-repeats=11 --dpi-desync-fooling=md5sig --dpi-desync-fake-tls="%~dp0tls_clienthello_www_google_com.bin" --new ^
--dpi-desync=fake,disorder2 --dpi-desync-autottl=2 --dpi-desync-fooling=md5sig

set SRVCNAME=zapret

net stop "%SRVCNAME%"
sc delete "%SRVCNAME%"
sc create "%SRVCNAME%" binPath= "\"%~dp0winws.exe\" %ARGS%" DisplayName= "zapret winws1" start= auto
sc description "%SRVCNAME%" "zapret"
sc start "%SRVCNAME%"

**Comment by CursendSoul** [comment]
> Чтобы сделать сервисом:
> 
> set ARGS=--wf-tcp=80,443 --wf-udp=443,50000-50099 ^ --filter-udp=50000-50099 --ipset="%~dp0ipset-discord.txt" --dpi-desync=fake --dpi-desync-repeats=6 --dpi-desync-any-protocol --dpi-desync-cutoff=n2 --new ^ --filter-udp=50000-50099 --new ^ --filter-udp=443 --hostlist="%~dp0list-youtube.txt" --dpi-desync=fake --dpi-desync-repeats=11 --dpi-desync-fake-quic="%~dp0quic_initial_www_google_com.bin" --new ^ --filter-udp=443 --dpi-desync=fake --dpi-desync-repeats=11 --new ^ --filter-tcp=80 --dpi-desync=fake,split2 --dpi-desync-autottl=2 --dpi-desync-fooling=md5sig --new ^ --filter-tcp=443 --hostlist="%~dp0list-youtube.txt" --dpi-desync=fake,split2 --dpi-desync-repeats=11 --dpi-desync-fooling=md5sig --dpi-desync-fake-tls="%~dp0tls_clienthello_www_google_com.bin" --new ^ --dpi-desync=fake,disorder2 --dpi-desync-autottl=2 --dpi-desync-fooling=md5sig
> 
> set SRVCNAME=zapret
> 
> net stop "%SRVCNAME%" sc delete "%SRVCNAME%" sc create "%SRVCNAME%" binPath= ""%~dp0winws.exe" %ARGS%" DisplayName= "zapret winws1" start= auto sc description "%SRVCNAME%" "zapret" sc start "%SRVCNAME%"

Это в блокнот и в .bat перевести?

**Comment by pepper-forms** [comment]
> > Чтобы сделать сервисом:
> > set ARGS=--wf-tcp=80,443 --wf-udp=443,50000-50099 ^ --filter-udp=50000-50099 --ipset="%~dp0ipset-discord.txt" --dpi-desync=fake --dpi-desync-repeats=6 --dpi-desync-any-protocol --dpi-desync-cutoff=n2 --new ^ --filter-udp=50000-50099 --new ^ --filter-udp=443 --hostlist="%~dp0list-youtube.txt" --dpi-desync=fake --dpi-desync-repeats=11 --dpi-desync-fake-quic="%~dp0quic_initial_www_google_com.bin" --new ^ --filter-udp=443 --dpi-desync=fake --dpi-desync-repeats=11 --new ^ --filter-tcp=80 --dpi-desync=fake,split2 --dpi-desync-autottl=2 --dpi-desync-fooling=md5sig --new ^ --filter-tcp=443 --hostlist="%~dp0list-youtube.txt" --dpi-desync=fake,split2 --dpi-desync-repeats=11 --dpi-desync-fooling=md5sig --dpi-desync-fake-tls="%~dp0tls_clienthello_www_google_com.bin" --new ^ --dpi-desync=fake,disorder2 --dpi-desync-autottl=2 --dpi-desync-fooling=md5sig
> > set SRVCNAME=zapret
> > net stop "%SRVCNAME%" sc delete "%SRVCNAME%" sc create "%SRVCNAME%" binPath= ""%~dp0winws.exe" %ARGS%" DisplayName= "zapret winws1" start= auto sc description "%SRVCNAME%" "zapret" sc start "%SRVCNAME%"
> 
> Это в блокнот и в .bat перевести?

я прост скопипастил preset_russia переименовал в сервис и заменил все содержимое этим

**Comment by ugarovterror** [comment]
до юза фикса выше в ветке вообще дс не работал, теперь заходит, но войс не работает, уверен, что проблема в МГТС, шарящие типы, подскажите пж что куда вписывать, буду благодарен!

**Comment by amirov400** [comment]
> до юза фикса выше в ветке вообще дс не работал, теперь заходит, но войс не работает, уверен, что проблема в МГТС, шарящие типы, подскажите пж что куда вписывать, буду благодарен!

Тоже самое. У меня тоже МГТС.. в дс заходит, но в голосовой канал не подключается.

**Comment by pepper-forms** [comment]
> до юза фикса выше в ветке вообще дс не работал, теперь заходит, но войс не работает, уверен, что проблема в МГТС, шарящие типы, подскажите пж что куда вписывать, буду благодарен!

ростелеком и билайн у кореша войс работает с этим фиксом

**Comment by amirov400** [comment]
> > до юза фикса выше в ветке вообще дс не работал, теперь заходит, но войс не работает, уверен, что проблема в МГТС, шарящие типы, подскажите пж что куда вписывать, буду благодарен!
> 
> Тоже самое. У меня тоже МГТС.. в дс заходит, но в голосовой канал не подключается.

Если у кого будет фикс для МГТС, напишите, пожалуйста.

**Comment by RealFoxes** [comment]
Господа вот рабочий скрипт, все текущие решение с перезапуском Windivert'а как по мне бред. Время от времени без обхода пропускался трафик, видимо что то до конца настраивают
```
start "zapret: discord" /min "%~dp0winws.exe" ^
--wf-l3=ipv4 --wf-tcp=443 --wf-udp=50000-50099 --filter-udp=50000-50099 --dpi-desync=fake --dpi-desync-cutoff=d4 --dpi-desync-repeats=6 --dpi-desync-any-protocol --new ^
--dpi-desync=fake,split2 --dpi-desync-fooling=badseq --dpi-desync-fake-tls=tls_clienthello_www_google_com.bin --hostlist="%~dp0list-discord.txt"
```

UPD: поменял рендж портов

**Comment by tesum** [comment]
> Чтобы сделать сервисом:
> 
> set ARGS=--wf-tcp=80,443 --wf-udp=443,50000-50099 ^ --filter-udp=50000-50099 --ipset="%~dp0ipset-discord.txt" --dpi-desync=fake --dpi-desync-repeats=6 --dpi-desync-any-protocol --dpi-desync-cutoff=n2 --new ^ --filter-udp=50000-50099 --new ^ --filter-udp=443 --hostlist="%~dp0list-youtube.txt" --dpi-desync=fake --dpi-desync-repeats=11 --dpi-desync-fake-quic="%~dp0quic_initial_www_google_com.bin" --new ^ --filter-udp=443 --dpi-desync=fake --dpi-desync-repeats=11 --new ^ --filter-tcp=80 --dpi-desync=fake,split2 --dpi-desync-autottl=2 --dpi-desync-fooling=md5sig --new ^ --filter-tcp=443 --hostlist="%~dp0list-youtube.txt" --dpi-desync=fake,split2 --dpi-desync-repeats=11 --dpi-desync-fooling=md5sig --dpi-desync-fake-tls="%~dp0tls_clienthello_www_google_com.bin" --new ^ --dpi-desync=fake,disorder2 --dpi-desync-autottl=2 --dpi-desync-fooling=md5sig
> 
> set SRVCNAME=zapret
> 
> net stop "%SRVCNAME%" sc delete "%SRVCNAME%" sc create "%SRVCNAME%" binPath= ""%~dp0winws.exe" %ARGS%" DisplayName= "zapret winws1" start= auto sc description "%SRVCNAME%" "zapret" sc start "%SRVCNAME%"

Нужно вставить в батник и запустить от админа, у меня с этим пресетом дропнулся плеер твича, убрал настройки для ютуба и заработало

**Comment by tesum** [comment]
> > > до юза фикса выше в ветке вообще дс не работал, теперь заходит, но войс не работает, уверен, что проблема в МГТС, шарящие типы, подскажите пж что куда вписывать, буду благодарен!
> > 
> > 
> > Тоже самое. У меня тоже МГТС.. в дс заходит, но в голосовой канал не подключается.
> 
> Если у кого будет фикс для МГТС, напишите, пожалуйста.

Работает для МГТС

**Comment by bol-van** [comment]
```
start "zapret: discord" /min "%~dp0winws.exe" ^
--wf-l3=ipv4 --wf-tcp=443 --wf-udp=1-65535 --filter-udp=50000-65535 --dpi-desync=fake --dpi-desync-cutoff=d4 --dpi-desync-repeats=6 --dpi-desync-any-protocol --new ^
--dpi-desync=fake,split2 --dpi-desync-fooling=badseq --dpi-desync-fake-tls=tls_clienthello_www_google_com.bin --hostlist="%~dp0list-discord.txt"`
```

Это трэш-конфиг. Изнасилование компа всем трафиком udp. Торентами utp в том числе.
Без какой-либо разумной цели.

**Comment by RealFoxes** [comment]
> ```
> start "zapret: discord" /min "%~dp0winws.exe" ^
> --wf-l3=ipv4 --wf-tcp=443 --wf-udp=1-65535 --filter-udp=50000-65535 --dpi-desync=fake --dpi-desync-cutoff=d4 --dpi-desync-repeats=6 --dpi-desync-any-protocol --new ^
> --dpi-desync=fake,split2 --dpi-desync-fooling=badseq --dpi-desync-fake-tls=tls_clienthello_www_google_com.bin --hostlist="%~dp0list-discord.txt"`
> ```
> 
> Это трэш-конфиг. Изнасилование компа всем трафиком udp. Торентами utp в том числе. Без какой-либо разумной цели.

Торентом не пользуюсь, и как уже понятно я не особо понимаю как настраивать скрипт. 
Так вот как можно починить этот конфиг под windows?

**Comment by ugarovterror** [comment]
> > > > до юза фикса выше в ветке вообще дс не работал, теперь заходит, но войс не работает, уверен, что проблема в МГТС, шарящие типы, подскажите пж что куда вписывать, буду благодарен!
> > > 
> > > 
> > > Тоже самое. У меня тоже МГТС.. в дс заходит, но в голосовой канал не подключается.
> > 
> > 
> > Если у кого будет фикс для МГТС, напишите, пожалуйста.
> 
> Работает для МГТС

это в preset_russia вставлять? у меня сразу крашит консольку

**Comment by sergey66u** [comment]
Ребята нашел решение проблемы 
[zapret-discord-youtube-1.5.1.zip](https://github.com/user-attachments/files/17578193/zapret-discord-youtube-1.5.1.zip)
ну распаковать и запустить дискорд от имени админа

Напишите кому помогло. Мне например помогло сразу, продайдер "Lovit"

**Comment by bol-van** [comment]
> Так вот как можно починить этот конфиг под windows?

Для начала надо перестать распространять плохие решения, потому что они разлетаются по сети, и насилует свой комп не 1 Вася, а 1000+
Здесь главная проблема, что в --wf перенаправляется трафик, который ни по каким профилям все равно не пройдет.
Перенаправление бесполезного трафика напрасно загружает процессор, не принося никакой пользы.
Для дискорда хватает обычно портов 50000-50099 udp
Написан же preset-russia. Он не работает, а ваш вариант - работает ?

**Comment by amirov400** [comment]
> > Так вот как можно починить этот конфиг под windows?
> 
> Для начала надо перестать распространять плохие решения, потому что они разлетаются по сети, и насилует свой комп не 1 Вася, а 1000+ Здесь главная проблема, что в --wf перенаправляется трафик, который ни по каким профилям все равно не пройдет. Перенаправление бесполезного трафика напрасно загружает процессор, не принося никакой пользы. Для дискорда хватает обычно портов 50000-50099 udp Написан же preset-russia. Он не работает, а ваш вариант - работает ?

Приветствую! есть решение под МГТС провайдера?

**Comment by RealFoxes** [comment]
> Написан же preset-russia. Он не работает, а ваш вариант - работает ?


Я использую старую версию(не помню точно версию качал в начале месяца), и дефолтный конфиг работает только на http часть дискорда, то есть голосовые каналы не работают. 
В новой версии что то поменялось в этом плане?

**Comment by bol-van** [comment]
> Приветствую! есть решение под МГТС провайдера?

Я не распространяю решения под провайдеры, тк не имею к ним доступа, да и следить за этим всем никаких сил не хватит и времени.
Но если кто-то постит что-то,  то либо это  надо делать грамотно, с пониманием как работает zapret. Либо хотя бы пишите приписку, что в конфиге не уверен, делал для себя, не принимайте как панацею.

> В новой версии что то поменялось в этом плане?

В последней версии bundle preset-russia.cmd сделан с учетом IP диапазонов discord и портов 50000-50099.
Минимально необходимое воздействие.
Сам его не проверял, тк не пользуюсь дискордом. Но исследования других товарищей и мои собственные исследования дампов трафика говорят в пользу этого варианта.

**Comment by RealFoxes** [comment]
@bol-van поменял в своём скрипте рендж портов

**Comment by ugarovterror** [comment]
> > > Так вот как можно починить этот конфиг под windows?
> > 
> > 
> > Для начала надо перестать распространять плохие решения, потому что они разлетаются по сети, и насилует свой комп не 1 Вася, а 1000+ Здесь главная проблема, что в --wf перенаправляется трафик, который ни по каким профилям все равно не пройдет. Перенаправление бесполезного трафика напрасно загружает процессор, не принося никакой пользы. Для дискорда хватает обычно портов 50000-50099 udp Написан же preset-russia. Он не работает, а ваш вариант - работает ?
> 
> Приветствую! есть решение под МГТС провайдера?

друг, у меня заработало с этим: https://github.com/Flowseal/zapret-discord-youtube
запустил service install и service remove, после general (МГТС) и все заработало, войс тоже

**Comment by FluFpery** [comment]
> Ребята нашел решение проблемы [zapret-discord-youtube-1.5.1.zip](https://github.com/user-attachments/files/17578193/zapret-discord-youtube-1.5.1.zip) ну распаковать и запустить дискорд от имени админа
> 
> Напишите, кому помогло. Мне, например, помогло сразу, продавец «Ловит»

Это работает?

**Comment by FluFpery** [comment]
Как удалить полностью старый запрет?

**Comment by amirov400** [comment]
> > > > Так вот как можно починить этот конфиг под windows?
> > > 
> > > 
> > > Для начала надо перестать распространять плохие решения, потому что они разлетаются по сети, и насилует свой комп не 1 Вася, а 1000+ Здесь главная проблема, что в --wf перенаправляется трафик, который ни по каким профилям все равно не пройдет. Перенаправление бесполезного трафика напрасно загружает процессор, не принося никакой пользы. Для дискорда хватает обычно портов 50000-50099 udp Написан же preset-russia. Он не работает, а ваш вариант - работает ?
> > 
> > 
> > Приветствую! есть решение под МГТС провайдера?
> 
> друг, у меня заработало с этим: https://github.com/Flowseal/zapret-discord-youtube запустил service install и service remove, после general (МГТС) и все заработало, войс тоже

Помогло, спасибо

**Comment by bedniSharek** [comment]
> Как удалить полностью старый запрет?

Как писал уже человек выше, если вы не можете удалить папку(мешает служба windivert), откройте диспетчер задач, остановите там winws.exe (если он там есть), откройте командную строку от имени администратора, и пропишите там:
sc delete windivert
sc stop windivert
sc qc windivert
после этого вы сможете спокойно удалить папку.

**Comment by mr1cloud** [comment]
Для мгтс, те кто испольхует dvtws:
```
dvtws --port 989 --daemon \
--filter-udp=443 --hostlist /opt/zapret/lists/general-list.txt --dpi-desync fake --dpi-desync-repeats 6 --dpi-desync-fake-quic /opt/zapret/files/fake/quic_initial_www_google_com.bin --new \
--filter-udp=50000-50100 --ipset /opt/zapret/lists/discord-ipset.txt --dpi-desync fake --dpi-desync-any-protocol --dpi-desync-cutoff d3 --dpi-desync-repeats 6 --new \
--filter-tcp=80 --hostlist /opt/zapret/lists/general-list.txt --dpi-desync fake,split2 --dpi-desync-autottl 2 --dpi-desync-fooling md5sig --new \
--filter-tcp=443 --hostlist /opt/zapret/lists/general-list.txt --dpi-desync fake --dpi-desync-autottl 2 --dpi-desync-repeats 6 --dpi-desync-fooling badseq --dpi-desync-fake-tls /opt/zapret/files/fake/quic_initial_www_google_com.bin
```

[discord-ipset.txt](https://github.com/Flowseal/zapret-discord-youtube/blob/main/ipset-discord.txt)
[general-list.txt](https://github.com/Flowseal/zapret-discord-youtube/blob/main/list-general.txt)

**Comment by grinchmm** [comment]
Это все хорошо. Поставил последнею версию  zapret-winws.
Запускаю preset_russia.cmd работает дискорд, ютуб, но при этом не открывается Авито. Кто знает как пофиксить, может что-то надо из скрипта удалить?

**Comment by Dzeret** [comment]
Парни а кто может мне помочь там в тг созвонимся а то не понимаю ничего

**Comment by karbonator55** [comment]
> > Как удалить полностью старый запрет?
> 
> Как писал уже человек выше, если вы не можете удалить папку(мешает служба windivert), откройте диспетчер задач, остановите там winws.exe (если он там есть), откройте командную строку от имени администратора, и пропишите там: sc delete windivert sc stop windivert sc qc windivert после этого вы сможете спокойно удалить папку.

После этого нужно что-то ещё делать, прописывать или перезапускать пк?

**Comment by RKN-cell** [comment]
> sc qc windivert

Скачал этот запрет,перестал работать рокстар гейм лаунчер, в чем проблема?

**Comment by HentayHentay** [comment]
> перестал работать рокстар гейм лаунчер

В античите. Давно известная проблема, но люди все равно идут и наступают на говно

**Comment by VladApoint** [comment]
Доброго время суток! Вопрос к знающим такой:
Проблемы с связью при использрвание Дискорда одновременно с Lesta Мир танков, выростает пинг связи.
есть у кого какое решение, или может кто подскажет что поправить.

**Comment by render69** [comment]
для linux что-то есть?

**Comment by TEPJO** [comment]
запрет отвалился, есть решение?

**Comment by TEPJO** [comment]
в msc надо офнуть killer и все заработает (как говорят) просто проблема в том что у меня нету этого, мало ли у кого то есть другой вариант

**Comment by RambilE** [comment]
у меня провайдер дом ру, запрет работал до прошлого воскресенья, и в тот же день перестали работать рандомные сайты без запрета (к примеру askubuntu), сегодня на время заработал этот же askubuntu и другие сайты и появилась возможность зайти в дискорд, но это проработало часа два, сейчас опять умерло

**Comment by root-localhost** [comment]
У меня на линухе всё работает, только что проверял )
